tools/pipelines/build-docs.yml

```diff
@@ -339,7 +334,9 @@ stages:
   displayName: 'Deploy website'
   pool: Small
   dependsOn: ['build', 'guardian', 'check_branch_version']
-  condition: eq(dependencies.check_branch_version.outputs['check_branch_version.SetShouldDeploy.shouldDeploy'], 'true')
+  condition: or(
```

Comment:
This still doesn't handle the case where `deployOverride` is set to `skip`. In that case we should _never_ deploy the site, even if it otherwise would.